che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/GitBasicAuthenticationCredentialsProvider.java

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - API 
+ *   SAP           - initial implementation
+ *******************************************************************************/
+package org.eclipse.che.git.impl.jgit;
+
+import org.eclipse.che.api.git.CredentialsProvider;
+import org.eclipse.che.api.git.GitException;
+import org.eclipse.che.api.git.UserCredential;
+import org.eclipse.che.api.git.server.dto.DtoServerImpls;
+import org.eclipse.che.api.git.shared.GitUser;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.inject.Singleton;
+
+@Singleton
+public class GitBasicAuthenticationCredentialsProvider implements CredentialsProvider {
+
+    private static final Logger LOG = LoggerFactory.getLogger(GitBasicAuthenticationCredentialsProvider.class);
+
+    public static final String GIT_PROJECT_ID = "git-project-id";
```

Comment:
Is it id of credentials provider? Maybe it would be better to change id to something like 'git-basic'